app/src/main/java/com/example/cmput301project/itemClasses/Photograph.java

```diff
@@ -1,4 +1,4 @@
-package com.example.cmput301project.ItemClasses;
+package com.example.cmput301project.itemClasses;
```

Comment:
Java naming condition dictates that all directory names should be in lower case.